python-checks/src/main/java/org/sonar/python/checks/cdk/PublicApiIsSecuritySensitiveCheck.java

```diff
@@ -20,23 +20,43 @@
 package org.sonar.python.checks.cdk;
 
 import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Predicate;
 import org.sonar.check.Rule;
+import org.sonar.plugins.python.api.tree.Expression;
+import org.sonar.plugins.python.api.tree.HasSymbol;
 
 import static org.sonar.python.checks.cdk.CdkPredicate.isString;
 import static org.sonar.python.checks.cdk.CdkUtils.getArgument;
 
 @Rule(key = "S6333")
 public class PublicApiIsSecuritySensitiveCheck extends AbstractCdkResourceCheck {
-  private static final String SAFE_API_MESSAGE = "Make sure that creating public APIs is safe here.";
+  private static final String ERROR_MESSAGE = "Make sure that creating public APIs is safe here.";
```

Comment:
IMO ERROR is not the right expression for this kind of rule/hotspot. Maybe `ISSUE_MESSAGE` or `MESSAGE` WDYT?

Reply:
fixed